.github/dependabot.yml

```diff
@@ -103,6 +103,10 @@ updates:
         patterns:
           - 'pandas'
           - 'pandas-stubs'
+      lxml:
+        patterns:
+          - 'lxml'
+          - 'lxml-stubs'
```

Comment:
Should put this on hold until we can see if the stubs package can be switched (see the readme here): https://github.com/lxml/lxml-stubs